.github/workflows/deploy_examples.yml

```diff
@@ -21,4 +21,4 @@ jobs:
       run: |
         echo "$PRIVATE_KEY" > private_key.pem
         chmod 600 private_key.pem
-        ssh -o StrictHostKeyChecking=no -i private_key.pem $USER@$HOST "cd /root/rooch && git pull origin main && bash scripts/deploy_examples.sh"
\ No newline at end of file
+        ssh -o StrictHostKeyChecking=no -i private_key.pem $USER@$HOST "cd /root/rooch && git pull origin main && docker pull ghcr.io/rooch-network/rooch:main && bash scripts/deploy_examples.sh"
```

Comment:
This CI workflow should not run when submitting PR because if the PR contains a break change, the deployment will fail.

Reply:
I merged this PR first. @yourmoonlight you can refactor it in new PR.